tests/data/metricvalue_repo1/testrun/metricvalue_verify_another/None/447/346/12345/metricvalue_verify_another_TestCam_447_346_12345_12_testrun.yaml

```diff
@@ -2,9 +2,9 @@
 blobs:
 - !<lsst.verify.Blob>
   data: {}
-  identifier: dfc508e8c4dc4309869f94fac556be45
+  identifier: adee78bab64e49b1bafa13369353a4dc
   name: verify.another
-identifier: cb3fe331870d48a2afdb8261b5d3e576
+identifier: 7f0507d5fcff49a89980f785eb05bf58
```

Comment:
Why are all these changes needed if you're just adding a metric???

Reply:
I regenerated the test repos to include a metric that only depends on the instrument, not visit/detector. It was easiest to just rebuild them from scratch with the relevant script.